docs/source-pytorch/accelerators/hpu_basic.rst

```diff
@@ -25,28 +25,6 @@ For more information, check out `Gaudi Architecture <https://docs.habana.ai/en/l
 
 ----
 
-Run on 1 Gaudi
```

Comment:
I thought these accelerator-specific sections were a bit repetitive. There's no logic here specific to the accelerator, and this is already described in the Trainer section. So I went ahead and removed it.

Any other docs suggestions are welcome

Reply:
I think the intention was more that it is self-contained and that you know that accelerator=hpu exists.

Reply:
Added back